BrowserStack.java

```diff
@@ -15,15 +14,17 @@
  *         isEmpty()    - return true if stack is not empty else false
  *
  */
+import java.util.*;
 public class BrowserStack {
 
-    String[] myStack;
+    ArrayList<String> myStack = null;
```

Comment:
**HINT:** :  Check if the stack is full and then double the size of the array by copying the contents of the old array to a new array of bigger size and then copy it back to old array (myStack)